### ISSUE TITLE
How to set Proxy from the Kameleo API?

### ISSUE THREAD

**Issue by JonathanR25** [kameleo-io/local-api-client-typescript#8]
What would be the proper code syntax form to set the Proxy details/credentials from the Kameleo API?

Also, could your provide some example code?

**Comment by MBailey01** [comment]
Below you can see the example of how to set up a proxy using the Local API Clients which you are free to use:

    ```
const createProfileRequest = BuilderForCreateProfile
      .forBaseProfile(chromeBaseProfileList[0].id)
      .setRecommendedDefaults()
      .setProxy(
        'socks5', # Options ('http', 'socks5', 'ssh')
        {
          host: '<proxy_host>',
          port: <proxy_port>,
          id: '<username>',
          secret: '<password>',
        },
      )
      .build();
    const profile = await client.createProfile({ body: createProfileRequest });
```

These answers are provided from the Kameleo [Set Proxy from API](https://help.kameleo.io/hc/en-us/articles/4455243268625-Set-Proxy-from-API) article.

**Comment by KameleoStaff** [comment]
We recently released Kameleo v4.0.0, which introduced several changes and improvements to the platform. As a result, this code has been updated to be fully compatible with the latest version.

```
@type {import('@kameleo/local-api-client').CreateProfileRequest} */
const createProfileRequest = {
    fingerprintId: fingerprints[0].id,
    name: "start with proxy example",
    proxy: {
        value: "socks5",
        extra: {
            host: process.env["PROXY_HOST"] || "",
            port: Number(process.env["PROXY_PORT"]) || 1080,
            id: process.env["PROXY_USERNAME"] || "",
            secret: process.env["PROXY_PASSWORD"] || "",
        },
    },
};
const profile = await client.profile.createProfile(createProfileRequest);
```

These answers are provided from the Kameleo [Set Proxy from API](https://help.kameleo.io/hc/en-us/articles/4455243268625-Set-Proxy-from-API) article.
